mlflow/utils/search_utils.py

```diff
@@ -187,7 +198,21 @@ def _invalid_statement_token(cls, token):
     def _process_statement(cls, statement):
         # check validity
         invalids = list(filter(cls._invalid_statement_token, statement.tokens))
-        if len(invalids) > 0:
+        if len(invalids) % 3 == 0 and len(invalids) != 0:
+            groupby_invalids = [invalids[i:i+3] for i in range(0, len(invalids), 3)]
+            invalid_clauses = ""
+            for invalid in groupby_invalids:
+                if invalid[1].value.lower() in ['like', 'ilike']:
```

Comment:
1. Define as constants (defined at the top of the file) rather than hard coding here
2. Prefer **UPPER** case for SQL keywords to distinguish from python code.

Reply:
Done